ch02/README.md

```diff
@@ -431,6 +431,8 @@ const int *const p3;    // illegal, p3 must initialize.
 const int *p;           // legal. a pointer to const int.
 ```
 
+[Here](ex2_28.cpp) is the code. It's good to see the errors reported by compiler!
+
 ##Exercise 2.29
```

Comment:
This is what I want . Because I see the comment is simple than the info provided by compiler. And for these errors ,the newbie should read the info provided by compiler , he can grow up!
If you think it is unused , I can keep it in my branch .